rpcd/playbooks/roles/rpc_post_upgrade/tasks/post-upgrade-swift-proxy.yml

```diff
@@ -0,0 +1,29 @@
+---
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: verify it's OK to check for errors, and that it won't cause
+# this playbook to fail in the case of failures happening under normal circumstances
+- name: Check swift md5s
+  shell: |
+    . {{ swift_venv_bin }}/activate
+    swift-recon --md5 | grep '[1-9][0-9]* error'
+  register: swift_recon
+  changed_when: swift_recon.rc != 0
+  failed_when: swift_recon.stdout_lines|length > 0
```

Comment:
This works, but will also not report failed if, for example, swift-recon isn't even installed.  I'm not sure it's worth re-jigging but it may be better to rely on swift-recon's exit code and then parse the output with ansible itself.